content/strategy-goals/strategy/growth-team/index.md

```diff
@@ -20,17 +22,27 @@ Our key metric is PQLs (Product Qualified Leads), as defined by (WIP):
+
+We have observed 4 different [user journeys](https://docs.google.com/document/d/1Jlt1oeOwjPC_PSPypEwnwfIkT6K2hw_Jek3J16ZiW1A/edit) (private). To support product-led, cloud-first growth we'll have to fix the following issues (and more we will discover):
 
-- The current initial experience is not developer-friendly as it requires users to self-host and/or talk to a salesperson to try out the product.
-- Setting up and onboarding Sourcegraph is painful and in many cases requires support from Sourcegraph's Customer Engineering and support team. In particular, the initial onboarding experience on cloud requires admins to read the docs.
-- There's no easy/practical way to invite collaborators and users to a cloud instance. Besides, the MAU caps (as determined by a user's license) limits the growth motion: a user may click on a Sourcegraph link and not be able to join.
-- Sourcegraph is massively used for collaboration in practice (sharing search query link), but there is no collaboration experience in-product. Several features such as Batch Changes (getting PRs merged), Notebooks, Monitors would deeply benefit from a better collaboration experience, that would power feedback loops.
+- **Signup and self-service activation**
+
+  - Setting up and onboarding Sourcegraph is high-commitment, and in many cases requires support from Sourcegraph's Customer Engineering. We observe that most signups don't add their codehost or activate on their own.
+  - It's hard and high commitment for users to add their code. It's frequent for trials to be in fact _pre-trials_: a potential champion requests a trial, and adds their personal code to build a case to convince their team or management to run a Sourcegraph trial. We need to make that motion very easy, both in-product, and by allowing those champions to self-serve what they need (security, one-pager, business case, etc) to drive a Sourcegraph trial.
+  - There are many friction points in the flow, that are likely to cause churn. For example, if initial users don't reset their password within the first 72 hours, they need to talk to a CE to get a new reset password link.
+  - The signup workflow is very manual, and relies on lots of ad hoc automation powered by Zapier that is brittle and constrains the experience. Managing trials requires lots of time from CE.
+
+- **Virality**
+  - There's no easy/practical way to invite collaborators and users to a cloud instance. Besides, unless SSO is setup and `allowSignup` is true, users can't join a pre-existing trial instance. That means unless the initial admins is very proactive, others won't join the instance.
```

Comment:
I think this flow would be awesome (and meet users' expectations):

- I sign in via Google Workspace or GitHub to start a trial on signup.sourcegraph.com
- If Google Workspace, my instance is preconfigured to allow other users to sign in via Google Workspace
- If GitHub, my instance is preconfigured to allow other users to sign in via GitHub

Reply:
Great point. This is the endgame and should be clearer in the page, updating.
Achieving this type of flow is one of the reasons the infrastructure [here](https://github.com/sourcegraph/handbook/pull/5717/files#diff-fdfa5563fd5508f35f1bbc523a056e5ec10ef625b7c4d7079ccf9efa6e654605R59) is super important. We want to be able to configure instances such as this, or adding OSS code by default (currently blocked: https://github.com/sourcegraph/sourcegraph/pull/44398)